src/doctor.rs

```diff
@@ -850,13 +838,12 @@ async fn accept(
         secret_key.public(),
         remote_addrs,
     );
-    if let relay_url = endpoint.home_relay().initialized().await? {
-        println!(
-            "\tUsing just the relay url:\niroh-doctor connect {} --relay-url {}\n",
-            secret_key.public(),
-            relay_url,
-        );
-    }
+    let relay_url = endpoint.home_relay().initialized().await?;
+    println!(
+        "\tUsing just the relay url:\niroh-doctor connect {} --relay-url {}\n",
+        secret_key.public(),
+        relay_url,
+    );
```

Comment:
Are you sure this is what you want, and not this?
```suggestion
    if let Some(relay_url) = endpoint.home_relay().get()? {
        println!(
            "\tUsing just the relay url:\niroh-doctor connect {} --relay-url {}\n",
            secret_key.public(),
            relay_url,
        );
    }
```